### DISCUSSION TITLE
进行数据备份时创建快照失败：OLAP_VERSION_ALREADY_MERGED, msg: make snapshot failed, version already merged

### DISCUSSION THREAD

**Discussion by leiyangcd** [apache/doris#12686] [A - General / Q&A]
Doris版本：0.15
备份SQL：BACKUP SNAPSHOT xx.SNAPSHOT_4
TO minio_repo
ON （xx);

错误信息：[90269: make_snapshot failed. status: -230]  /  [OLAP_VERSION_ALREADY_MERGED, msg: make snapshot failed, version already merged]

**Comment by yuanyuan8983** [comment]
You can add me to WeChat: lyy0222. You can send me your questions and let me have a look. I can also bring you to the Doris community group, so that you can solve problems better later.
